src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3241,15 +3242,33 @@ static size_t large_page_init_decide_size() {
     return 0;
   }
 
-#if defined(IA32) || defined(AMD64)
-  if (size > 4*M || LargePageSizeInBytes > 4*M) {
-    WARN("JVM cannot use large pages bigger than 4mb.");
-    return 0;
-  }
+  // Check if OS version is 11 or greater
+  if (schedules_all_processor_groups) {
```

Comment:
either have saint change the name to something more generic or call instead "win32::is_windows_11_or_greater() || win32::is_windows_server_2022_or_greater()"